### DISCUSSION TITLE
What Is the Keybind for the "Super Hero Fly" Option to shoot lasers?

### DISCUSSION THREAD

**Discussion by mrnasty0306** [YimMenu/YimMenu#3122] [Q&A]
I saw some guy in my lobby today shoot lasers from his eyes yet I can't seem to find the keybind to activate it. It's activated in the menu but no button seems to do it.

**Comment by Rxann** [comment]
The menu does not have such a thing. It is probably a 3rd party script for a P2C like Stand.

**Reply by mrnasty0306** [reply]
![pic2](https://github.com/YimMenu/YimMenu/assets/169870817/af2d5f1a-a640-4caf-8fad-f4fda76cf8e3)

**Reply by Rxann** [reply]
> ![pic2](https://github.com/YimMenu/YimMenu/assets/169870817/af2d5f1a-a640-4caf-8fad-f4fda76cf8e3)
> 
> 

Hm, I don't really use this but try pressing the trigger or something

**Reply by GreySideMM** [reply]
There seems to be no lasers or other shooting inside the code at all, see:
https://github.com/YimMenu/YimMenu/blob/7f8977adcf28da3ff60710249664cd4935452b4a/src/backend/looped/self/super_hero_fly.cpp
Idk if it got removed or why it is in its description.

**Reply by Rxann** [reply]
Yes, I agree
